sphinx_gherkindoc/writer.py

```diff
@@ -368,7 +368,8 @@ def table(table: behave.model.Table, inline: bool = False) -> None:
     # but not having scenarios in it,
     # only exclude the feature (and its description, etc)
     # if include/exclude logic has been activated.
-    if not included_scenarios and (include_tags or exclude_tags):
+    # or in the event that there is an empty feature file.
+    if not included_scenarios:
```

Comment:
I don't like confuscating the case of an empty feature file with filtering scenarios by tags. We should know earlier in the code if the feature file is empty or not. Really this is changing existing behavior in addition to fixing the issue you are having.

I think the right place to handle this is above in the `feature = feature_class(root_path, source_path)` line (364). Based on your error in #50, `feature` is `None`. But there is a separate case where the file is not empty, but it doesn't have any scenarios (`feature` is not `None`, but `feature.scenarios` is `[]`), like this:

```gherkin
Feature: No Scenario

```

In that case, we still want to document that the feature is there, it just won't have any scenario documented, which makes sense. Sort of like if you have a function with only `pass` in it, Sphinx, would still document it.

So I think the right approach is something like this above:

```py
feature = feature_class(root_path, source_path)
# If the contents of the file do not allow a feature to be parsed,
# do not consider the file a feature file, and exclude it from the docs.
if feature is None:
    return None
```

Now, in your case, if you still want to see the feature file in the docs, you could just comment out all the scenarios (I think that is better), and if you comment the whole file out, it will just be ignored. But we aren't ignoring features that just don't have scenarios.

Based on all of that, I'd love to see test cases for:
- A completely empty file with an extension that isn't `.feature` or `.rst`
- A completely empty file with a `.feature` extension
- A file with a `.feature` extension, `Feature: Something` as the first line, but no scenarios
- A file with a `.feature` extension, `Feature: Something` as the first line, with scenarios, but all commented out (your case from #50)